packages/snaps-cli/src/cmds/build/bundle.test.ts

```diff
@@ -4,6 +4,7 @@ import { bundle } from './bundle';
 import * as bundleUtils from './bundleUtils';
 
 jest.mock('browserify');
+jest.mock('babelify', () => 'mockBabelify');
```

Comment:
The default export of `babelify` is actually a function, but we just need to know that whatever is exported under that name is passed as the first argument to the transform in the test cases in this file.